lib/rspec/core.rb

```diff
@@ -31,6 +31,7 @@
   option_parser
   configuration_options
   runner
+  profiler
```

Comment:
Originally we had both profilers in that file, which would allow a cost reduction when not using `--profile` but in order to make `config#profile_examples` dynamic we must always have the profiler on (it's not actually doing much just capturing stats that aren't captured during run time normally)